app/templates/cli.js

```diff
@@ -1,9 +1,13 @@
 #!/usr/bin/env node
-'use strict';
+'use strict';<% if (babel) { %>
+import meow from 'meow';
+import <%= camelModuleName %> from './';
+
+const cli = meow([<% } else { %>
 var meow = require('meow');
 var <%= camelModuleName %> = require('./');
 
-var cli = meow([
+var cli = meow([<% } %>
```

Comment:
My concern was that with:

```js
meow(`
    Usage
`);
```
`Usage` is indented 4 spaces in the input string.

But with

```js
meow([
  'Usage'
])
```

There is no whitespace before `Usage`. 

`redent` is smart enough to figure the whole thing out.

It is important you not have text on the first line of your template though:

```js

// good
meow(`
    Usage
        blah blah blah
`);

// bad
meow(
    `Usage
         blah blah blah
`);
```